README.md

```diff
@@ -1,78 +1,189 @@
-# Kiryu Coco Gradution
-It was not long enough, and we will miss you Coco. Congratulations on your graduation.
-
-# Credits
-
-## 2D2C Collaborators
-
-ChienHS
-
-Nyanako [NanakoTW]
-
-## Website Contributors
-
-DuongTyler (Frontend, Backend, Devops)
-
-Zapple (Frontend, Backend)
-
-LrJim (Frontend, Backend)
-
-Laska (Frontend)
-
-Ringo (Frontend)
-
-## Project Creators and Leads
-
-XOF ("Dragon Heart" Game and original arts collection)
-
-Capt-Jules (Digital scrapbook)
-
-Revel (Original music for Coco)
-
-PheelGagg (Letters to Coco)
-
-jtron (Farewell song for Coco)
-
-BadGrammar (Coco Dance Animation)
-
-Kelanduo (World-level Giant Minecraft Fanart)
-
-## Art and Design Contributors
-
-nicooooooo (Coco brooch and asacoco tail assets)
-
-[FriedPeanuts (Coco star hairpin asset)](https://twitter.com/PistachiosChips)
-
-Rara (Diploma design and webpage tiling)
-
-[Mr. Cres (Site background)](https://www.pixiv.net/en/users/25645372)
-
-Guy (Color consultant)
-
-## Coordinators and Other Awesome Peoples in the Shadows
-
-Ice
-
-RazorNaitor
-
-Necrosis
-
-VintageGhost
-
-Kiwi & Hiroki (Community Message)
-
-# Join the MASS Discord
-
-https://discord.gg/yTY9rdp
-
-
-# Join the 2D2C Discord
-
-https://discord.gg/whnHahqQeg
-
-
-# Contact Us
-
-Did we forget to credit you? Found a bug on the website? Contact us at manotomo[at]googlegroups.com
-
-Server issue? Contact us at webmaster[at]manotomo.com
+# Mano Aloe Messages of support website
+
+Website for the messages of support from the [Mano Aloe Support Squad (M.A.S.S)](https://discord.gg/Y4BBfyM) discord server
+
+## Contributors
+
+### Website Team
```

Comment:
Should add LrJim since he did a lot of the work for revamping the frontend.

Reply:
Wait oh yea I just copy pasted the old credits thingy... Will do that in a sec